plugins/modules/silence_info.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Paul Arthur <paul.arthur@flowerysong.com>
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'XLAB Steampunk'}
+
+DOCUMENTATION = '''
+module: silence_info
+author:
+  - Paul Arthur (@flowerysong)
+  - Aljaz Kosir (@aljazkosir)
+  - Manca Bizjak (@mancabizjak)
+  - Tadej Borovsak (@tadeboro)
+short_description: Lists Sensu silence entries
+description:
+  - For more information, refer to the Sensu documentation at
+    U(https://docs.sensu.io/sensu-go/latest/reference/silences/)
+extends_documentation_fragment:
+  - sensu.sensu_go.auth
+options:
+  subscription:
+    description:
+      - The name of the subscription the entry should match. If left empty a silencing entry will
+        contain an asterisk in the subscription position.
+    default: "*"
+    type: str
+  check:
+    description:
+     - The name of the check the entry should match. If left empty a silencing entry will contain an
+       asterisk in the check position.
+    default: "*"
+    type: str
+'''
+
+EXAMPLES = '''
+- name: List Sensu silence entries
+  silence_info:
+  register: result
+'''
+
+RETURN = '''
+objects:
+  description: list of Sensu silence entries
+  returned: always
+  type: list
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.sensu.sensu_go.plugins.module_utils import (
+    arguments, errors, utils
+)
+from ansible_collections.sensu.sensu_go.plugins.modules.silence import name_is_valid
```

Comment:
This will not work. Ansiballz framework only bundles things from the `module_utils` package.

Reply:
oh... of course :sweat_smile: